src/ui/common/src/handlers/v2/NodeMetricResultContentGet.ts

```diff
@@ -0,0 +1,20 @@
+// This file should map exactly to
+// src/golang/cmd/server/handler/v2/node_metric_result_content_get.go
+
+import {
+  NodeArtifactResultContentGetRequest,
+  NodeArtifactResultContentGetResponse,
+} from './NodeArtifactResultContentGet';
+
+export type NodeMetricResultContentGetRequest =
+  NodeArtifactResultContentGetRequest;
+
+export type NodeMetricResultContentGetResponse =
+  NodeArtifactResultContentGetResponse;
+
+export const nodeMetricResultContentGetQuery = (
+  req: NodeMetricResultContentGetRequest
+) => ({
+  url: `workflow/${req.workflowId}/dag/${req.dagId}/node/metric/${req.nodeId}/result/${req.nodeResultId}/content`,
```

Comment:
Instead of the route being ```node/metric/${req.nodeId}```

Should we instead do ```node/${req.nodeId}/metric``

cc @likawind for more comments here.

Reply:
hmmm I feel ID follow after type sounds correct? Type is a larger set and it makes sense to happen before?